### PR TITLE
Rbac docs first draft.

### PR DESCRIPTION
**Pull request by VeljkoMaksimovic**

### CODE REVIEW

File: docs/security/rbac-authorization.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+---
+
+# Rbac model
+
+Semaphore 2.0 uses a role-based access control model for user authorization. This page will document all existing permissions, roles,
+and how to manage them.
+
+## Roles on Semaphore
+
+All of the roles (and permissions) within the Semaphore are devided into <i>organization-level</i> and <i>project-level</i>.
```
Comment (radwo) [inline docs/security/rbac-authorization.md:12]: We are using markdown, not HTML, for documentation.

**Review by radwo — CHANGES_REQUESTED** [review]

File: docs/security/rbac-authorization.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+</tr>
+</tbody>
+</table>
+
+## Role Management
+
+Users can get roles in a few different ways (from different sources):
+<ul>
+<li>Each user can get one organization role assigned to them directly (from the organization's Admin or Owner).</li>
+<li>User can get an organization-level role from membership within one (or more) organization groups.</li>
```
Comment (radwo) [inline docs/security/rbac-authorization.md:195]: We do not have groups yet.

File: docs/security/rbac-authorization.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+</ul>
+
+Within each project user can be assigned a role:
+<ul>
+<li>Directly form projects Admin or Maintainer.</li>
+<li>From membership within one (or more) organization groups.</li>
+<li>From access rights within [project's repository](#repository-to-role-mappings) (GitHub or BitBucket).</li>
```
Comment (radwo) [inline docs/security/rbac-authorization.md:202]: Please remove information about specific repository providers

File: docs/security/rbac-authorization.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+### Organization role to project role mappings
+Having specific roles within the organization can grant you a role within all of the projects automatically.
+
+Organization <b>Owner</b> -> Project <b>Admin</b>
+<br/>
+Organization <b>Admin</b> -> Project <b>Admin</b>
+<br/>
+
+# Custom roles
```
Comment (radwo) [inline docs/security/rbac-authorization.md:228]: We also do not have this currently.

Comment (VeljkoMaksimovic) [reply docs/security/rbac-authorization.md:233]: Ok. I put this here in the draft for the purpose of conveying a full picture of what is RBAC intended to be.

File: docs/security/custom-roles.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+Description: Custom roles for Semaphore platform.
+---
+
+# Custom Roles
+
+If your organization needs more roles where permissions would be assigned with 
+higher granularity, you can define custom roles that will be visible only
```
Comment (radwo) [inline docs/security/custom-roles.md:8]: Maybe we can skip the `that will be visible only within your organization.` part? It should be obvious :)

Comment (VeljkoMaksimovic) [reply docs/security/custom-roles.md]: Done

**Comment by lucianoliberti** [comment]
@VeljkoMaksimovic please don't merge this doc until it's reviewed by the [Copy Writer](https://github.com/Charismatron). As soon as he has access I'll add him as a reviewer. Thanks

**Comment by VeljkoMaksimovic** [comment]
> @VeljkoMaksimovic please don't merge this doc until it's reviewed by the [Copy Writer](https://github.com/Charismatron). As soon as he has access I'll add him as a reviewer. Thanks
@lucianoliberti No problem

**Review by Charismatron — APPROVED** [review]
I've made pull requests for changes to be merged.

**Review by radwo — APPROVED** [review]

**Pull request merged**